common/config/azure-pipelines/ci-ios.yaml

```diff
@@ -44,10 +44,10 @@ jobs:
       - checkout: self
         clean: true
 
-      - task: NodeTool@0
+      - task: UseNode@1
         displayName: "Use Node 20"
         inputs:
-          versionSpec: 20.x
+          version: 20.x
```

Comment:
this one we can leave at 20 since @tcobbs-bentley is still working on node 22 support for mobile

Reply:
It's worth mentioning that the build-time Node version on the build host (Mac) doesn't need to match the runtime Node version on the device (iOS/Android). So using Node 22 for the iOS build is probably fine.